middlewares/authentication.go

```diff
@@ -0,0 +1,19 @@
+package middlewares
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	session "github.com/gutakk/go-google-scraper/helpers/session"
+)
+
+func EnsureNoAuthentiction(c *gin.Context) {
```

Comment:
Typo:

```suggestion
func EnsureNoAuthentication(c *gin.Context) {
```

Reply:
Fixed in 1ced9555950fdcc901e965496eddad3251270cee